plugins/headers.rst

```diff
@@ -0,0 +1,143 @@
+Header Plugins
+==============
+
+Header plugins are useful to manage request headers. Many operations are possible:
+
+Default headers values
+----------------------
+
+The plugin Http\Plugins\HeaderDefaultPlugin allows to set default values for given headers.
```

Comment:
i like the way joel is doing this in #76, with ``HeaderDefaultPlugin`` (local class in double backticks to make it shown as code. FQN in the code example is enough